### DISCUSSION TITLE
Mapping setup

### DISCUSSION THREAD

**Discussion by MehdiBigBoss** [mnit-rtmc/iris#165] [General]
Hello

The tutorial in the site https://mnit-rtmc.github.io/iris/mapping.html, explaining how generate maps for iris is obsolete.
A lot of files required are no longer available on internet.

Can you update this part please.

Thank you in advance.

**Comment by gordonparikh** [comment]
Hi @MehdiBigBoss,

I created updated instructions that you can find on our wiki [here](https://github.com/SRF-Consulting-Group-Inc/iris/wiki/Generating-OSM-Tiles). I'll note that since IRIS switched to nginx recently, you'll either need another server to do this, as I covered briefly in the guide, or use a dual Apache/nginx setup somehow.

If you have questions or run into issues I'll try to help where I can. Good luck!

Best,
Gordon

**Comment by AndrewMC6** [comment]
I'm having trouble getting the guide above to work successfully, according to some searching around there are a number of elements in the scripts that are now broken. Still not able to create tiles for my iris installation. Nor can I seem to figure out how to get a static PNG to work with the server

**Comment by gordonparikh** [comment]
Hi @AndrewMC6,

Are you talking about the original IRIS guide? Or the [guide](https://github.com/SRF-Consulting-Group-Inc/iris/wiki/Generating-OSM-Tiles) I mentioned in my previous post? If you haven't looked at it yet, I'd suggest taking a look at the guide I posted and seeing if you can get anywhere. It may not work perfectly for your situation, but it should at least get you a little further than MNIT's guide, which is out of date.

If you are talking about the guide I posted, let me know where you're having issues and I can try to help.

Thanks,
Gordon

**Comment by AndrewMC6** [comment]
@gordonparikh I was using the guide in your previous post, I'd spun up a VM to just host ubuntu as a mapping server. Ran your guide and found there were several errors, I'd have to run it again to see exactly what they are. Regardless I'm still not able to get iris to give me a glimmer of a map.

**Comment by AndrewMC6** [comment]
Hi all, 

Has there been any traction on updating the mapping with iris? Has anyone recently been able to create a map server?

**Comment by gordonparikh** [comment]
Hi Andrew,

Sorry, I have been meaning to look into it but haven't had the time. I will need to in the next month or so for a new effort that's starting up though, so I should have more information soon and will keep you posted.

Thanks,
Gordon

**Comment by AndrewMC6** [comment]
> Hi Andrew,
> 
> Sorry, I have been meaning to look into it but haven't had the time. I will need to in the next month or so for a new effort that's starting up though, so I should have more information soon and will keep you posted.
> 
> Thanks, Gordon

Gordon,

If you need any assistance testing, or anything like that, let me know and I'll see if I can help. I have iris itself running, but mapping is the only thing holding me back from testing it in earnest.

**Comment by AndrewMC6** [comment]
Gordon or MN-IT any progress on this? This is a major setback when installing iris. There might be a way forward using bing maps? I believe for free also.

**Comment by gordonparikh** [comment]
Sorry for the wait, my work was delayed a bit but I will be getting to it very soon and will make some updates to the process then. It's possible there are alternatives using other map services as well, though I haven't personally researched these and am not sure about the licensing implications for free use. Could be interesting though.
